go.mod

```diff
@@ -16,6 +16,7 @@ require (
 	github.com/hashicorp/go-retryablehttp v0.6.8
 	github.com/joho/godotenv v1.4.0
 	github.com/labstack/echo/v4 v4.9.0
+	github.com/prometheus/client_golang v1.11.0
```

Comment:
Is it possible to bump this to v1.11.1 instead? That version seems to have a high vulnerability in it. 
![Screenshot 2023-05-12 at 10 30 02](https://github.com/harness/ff-proxy/assets/42169772/8e40e34f-2e9e-48ec-b67d-e0ac343810f6) . We can probably change this scan to fail the build if any highs exist now we're pretty clean

Reply:
Yep will do